FRE-nctools_documentation_github.txt

```diff
@@ -0,0 +1,273 @@
+FRE-nctools Documentation Collection For GIthub
+
+FRE-NCtools is a collection of tools to help with the creation and
+manipulation of netCDF files used or written by the climate
+models.
+
+Available in FRE-NCtools:
+
+### Grid Tools Complete Description
+
+1.Check_mask
+
+   check_mask is used to configure the processors which contains all land points to be    
+   masked out for ocean model (--model ocean). It also is used to configure the           
+   processors which contains all ocean points to be masked out for land model             
+   (--model land). This program is supposed to run on single processor. This tool will    
+   print out number of processors to be masked, layout of the domain, masked out          
+   processors list. These information will be written into files, with file name is       
+   mask_table.n_mask.layout(1)Xlayout(2). The model is expected to read file mask_table   
+   if you want to mask out some processors.                                               
+                                                                                          
+   The ocean model is assumed to be logically rectangular grid and land model could be    
+   multiple tiles grid (like cubic shpere grid with 6 tiles). For land model, each        
+   processor list has three entries: layout position in x-direction, layout position in   
+   y-direction and tile number. For ocean model, each processor list has two entries:     
+   layout position in x-direction, layout position in y-direction.                        
+                                                                                          
+   The purpose of using mask domain is to decrease the processor usage without            
+   decreasing the performance, without changing bits relative to the case run without the 
+   mask domain option. The mask domain option acts to mask out regions that have zero     
+   ocean points (i.e., points that are all land) in both the ocean and sea ice models or  
+   mask out regions that have zero land points (i.e., points that are all ocean) in land  
+   model                                   
+
+2.Make_Remap_File
+
+  This program creates remap file from Cubic sphere grid to Cubic sphere grid.

```

Comment:
I wish there was first a definition for "remap file". I don't know what it is!

Reply:
Yes a definition for the remap file would help for sure.